tests/src/lib.rs

```diff
@@ -49,4 +69,12 @@ mod tests {
         assert_eq!(token.balance_of(JOE), amount);
         assert_eq!(token.allowance(ALI, BOB), allowance - amount);
     }
+
+    #[test]
+    #[should_panic]
+    fn test_erc20_transfer_from_too_much() {
+        let amount = ERC20_INIT_BALANCE + 1;
```

Comment:
The amount can be `1` as well.

Reply:
The amount that initialized within the contract is the INIT balance, If the amount is 1, then the transfer actual goes through.

Reply:
No it's not.

Reply:
There was an error in the test, I wrote. Fixed it.